projects/WTI-API/src/main/controllers/ContestController.java

```diff
@@ -566,7 +582,53 @@ public Response getStandings(
 					// we got the internal contest; pass it to the DefaultScoringAlgorithm and get back updated standings
 					try {
 						Properties props = internalContest.getContestInformation().getScoringProperties();
-						String xmlStandings = dsa.getStandings(internalContest, props, logger);
+
+						xlog(logger, "debug 22 scoreboard 2 props "+props);
+						if (props != null) {
+						       Set<Object> set = props.keySet();
+
+						        String[] keys = (String[]) set.toArray(new String[set.size()]);
+
+						        Arrays.sort(keys);
+								xlog(logger, "debug 22   Scoring Properties, there are " + keys.length + " keys");
+
+						        for (String string : keys) {
+									xlog(logger, "debug 22      " + string + "='" + props.get(string) + "'");
+						        }
+						}
+						
+						props = DefaultScoringAlgorithm.getDefaultProperties();
+						xlog(logger, "debug 22  set Default Scoring Props for "+key);
+
+						
+						/**
+						 * Should we use filer by division logic to filter runs ?
```

Comment:
Please fix typo in the above statement:  "filer" should be "filter"?

Reply:
rewrote comment/description of useDivisionFilter

Reply:
pushed

Reply:
rewrote comment/description of useDivisionFilter